### PR TITLE
chore: postpone devnet KromaMPT time

### PR DESCRIPTION
**Pull request by Pangssu**
1735794000 (2025-01-02 5:00:00 AM UTC)

### CODE REVIEW

File: params/superchain.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ var KromaChainConfigs = map[uint64]*KromaChainConfig{
 	KromaDevnetChainID: {
 		CanyonTime:   uint64ptr(1707292800),
 		EcotoneTime:  uint64ptr(1712908800),
-		KromaMPTTime: uint64ptr(1735189200),
+		KromaMPTTime: uint64ptr(1735534800),
```
Comment (sm-stack) [inline params/superchain.go:35]: I think the time should be located at the point after the upgrade to v0.6.0, maybe Tuesday 2pm or Thursday 2pm.

Comment (Pangssu) [reply params/superchain.go:35]: Okay then I think Thursday seems like a good option.

**Comment by 0xbenyun** [comment]
LGTM, but the PR description is incorrect.

> (2024-01-02 5:00:00 AM UTC)

```suggestion
 (2025-01-02 5:00:00 AM UTC)

**Review by kangsorang — APPROVED** [review]

**Review by sm-stack — APPROVED** [review]
LGTM

**Review by 0xbenyun — APPROVED** [review]

**Review by seolaoh — APPROVED** [review]

**Pull request merged**